docs/book/v2/links-and-resources.md

```diff
@@ -198,4 +198,42 @@ $resource = $resource->withLink($link);
 $resource = $resource->withoutLink($link);
 ```
 
+To maintain consistency in the structure of the response, you may choose to embed both non-empty and empty collections within the `_embedded` section. This can be achieved by enabling the `embed-empty-collections` configuration option.
+
+To enable this feature, modify the configuration file as follows:
```

Comment:
_Which configuration file?_

Here is some more information on this topic:

## Filenames

A filename must be placed before a code example because there are always two parts for this information: 

1. what kind of file is this
2. and what is the name or path.

Both informations in a code comment would be too long and misplaced.

#### Examples

> Create a file for the module configuration called `module.config.php` under `laminas-mvc-tutorial/module/Album/config/`:

> …in a factory, e.g. `src/Album/Controller/AlbumControllerFactory.php`:

> …in a module configuration file, e.g. `module/Album/config/module.config.php`:

Reply:
sure, but that's not consistent with https://docs.mezzio.dev/mezzio-hal/v2/cookbook/generating-custom-resources